.github/workflows/anchor_containerized_test.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow checks out code, performs an Anchore container image
+# vulnerability and compliance scan, and integrates the results with
+# GitHub Advanced Security code scanning feature.  For more information on
+# the Anchore scan action usage and parameters, see
+# https://github.com/anchore/scan-action.  For more information on
+# Anchore container image scanning in general, see
+# https://docs.anchore.com.
+
+name: Anchore Container Scan
+
+on: push
+
+jobs:
+  Anchore-Build-Scan:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout the code
+      uses: actions/checkout@v2
+    - name: Change working dir
```

Comment:
I don't think this is named correctly, `pwd && ls` doesn't do any directory changes

Reply:
Removed debug step and re-commited the file